ApollosProject/apollos-prototype
# Remove inactive or archived groups

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR fixes the group data source so that it actually removes any groups that are inactive or archived.

### What design trade-offs/decisions were made?
None.

### How do I test this PR?
Essentially you want to make sure that you have a group that you are a member of that is inactive or archived. Then make sure that said group doesn't show up when running `getByPerson` (you can run that by running the `getCurrentUserProfile` query and making sure that you're pulling `isGroupLeader`). 

FWIW, I copied this over and tested on NewSpring since I cannot get the core app to run anymore. 🤷‍♂️

### What automated tests did you write?
None. But I updated snapshots.

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
